metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1769,11 +1753,13 @@ def _exec_feature_query(self, *, feature_key: str) -> Dict:
 
```

Comment:
would this be a good time to remove owner tags? I think it is worth removing and re-introducing if and when we want an `Owner_Tag` entity.